images/tool/Dockerfile

```diff
@@ -16,7 +16,10 @@ FROM golang:1.20.2
 
 # Required input: -e TOOL_NAME=name, i.e. TOOL_NAME=flaky-test-reporter
 ARG TOOL_NAME
+ARG COMMIT_HASH
 
 RUN GOBIN=/tools go install "knative.dev/toolbox/${TOOL_NAME}@latest"
+RUN ln -s "/tools/${TOOL_NAME}" /bin/knative-tool
+ENV COMMIT_HASH=COMMIT_HASH
 
-ENTRYPOINT ["/tools/${TOOL_NAME}"]
+ENTRYPOINT ["/bin/knative-tool"]
```

Comment:
```suggestion
ENTRYPOINT ["/tools/${TOOL_NAME}"]
```

Reply:
This will not work. Even if the TOOL_NAME be defined as env var.

`/bin/echo $VAR` isn't same as `/bin/$VAR`. The `$VAR` isn't expanded, so the `/bin/echo $VAR` can work because `echo` binary is executed, and it then looks into environment variables. `/bin/$VAR` will try to execute exactly `/bin/$VAR`, which doesn't exist.

Reply:
Hmm i see